### PR TITLE
hyungro lee's profile

### PR DESCRIPTION
**Pull request by lee212**

### CODE REVIEW

File: _people/hyungro_lee.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+name: "Hyungro Lee"
+collection: people
+avatar: hyungro.jpg
```
Comment (iparask) [inline _people/hyungro_lee.md:4]: Can you add your picture under `images/people`? Thank you!

File: _people/hyungro_lee.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+joined: 2018
+position: researcher
+permalink: /people/lee212
+---
```
Comment (iparask) [inline _people/hyungro_lee.md:8]: Would you mind changing the permalink to `first_last` name or something similar?

**Review by iparask — CHANGES_REQUESTED** [review]

File: _people/hyungro_lee.md (hunk 1 above)
Comment (lee212) [reply _people/hyungro_lee.md:4]: added 3aa3512bed76db3f6c12283e315626871a95acf9

File: _people/hyungro_lee.md (hunk 2 above)
Comment (lee212) [reply _people/hyungro_lee.md:8]: changed 032d36dc9d9a04cc7a271af448c8a80cf3e93340

**Review by iparask — APPROVED** [review]

**Pull request merged**